javalin/src/main/java/io/javalin/plugin/rendering/vue/JavalinVue.kt

```diff
@@ -14,14 +14,15 @@ import java.nio.file.Files
 import java.nio.file.Path
 import java.nio.file.Paths
 import java.util.stream.Collectors
+import kotlin.reflect.KClass
 import io.javalin.plugin.rendering.vue.JavalinVue.resourcesJarClass as jarClass
 
 object JavalinVue {
     // @formatter:off
     internal var isDev: Boolean? = null // cached and easily accessible, is set on first request (can't be configured directly by end user)
     @JvmField var isDevFunction: (Context) -> Boolean = { it.isLocalhost() } // used to set isDev, will be called once
     @JvmField var optimizeDependencies = true // only include required components for the route component
-    @JvmField var resourcesJarClass = PathMaster::class // can be any class in the jar to look for resources in
+    @JvmField var resourcesJarClass:KClass<*> = PathMaster::class // can be any class in the jar to look for resources in
```

Comment:
```suggestion
    @JvmField var resourcesJarClass: Class<*> = PathMaster::class.java // can be any class in the jar to look for resources in
```

Reply:
You are completely right!
I use Kotlin for one and a half year and sometimes forget to think backward compatibility toward Java.